pkg/interface/publish/src/js/components/lib/new-post.js

```diff
@@ -98,7 +100,7 @@ export class NewPost extends Component {
             popout={props.popout}
           />
           <button
-            className="v-mid w-100 mw7 tl pl4 h1"
+            className={"v-mid w-100 mw7 tl h1"+submitButtonPadding}
```

Comment:
You’ll need a space in one of these strings or it will concatenate the strings directly and make a monstrous monoclass

Reply:
There is a space in submitButtonPadding though

Reply:
Oh, sorry you’re right. I saw it wrong x_x

Reply:
I just realized this change only looks good when the screen width is small enough, otherwise it looks bad. Going to revert this, not sure what the right solution is, but it shouldn't have been in this PR in the first place.